packages/analytics/analytics-metric-provider/package.json

```diff
@@ -56,13 +56,17 @@
     "swrv": "^1.0.4"
   },
   "peerDependencies": {
+    "@kong-ui-public/analytics-config-store": "workspace:^",
     "@kong-ui-public/analytics-utilities": "workspace:^",
     "@kong-ui-public/i18n": "workspace:^",
-    "@kong/kongponents": "9.0.0-alpha.107"
+    "@kong/kongponents": "9.0.0-alpha.107",
+    "pinia": "^2.1.7"
```

Comment:
Sadly, it needs the dependency because it creates Pinia instances in the sandbox and in the test harness.  I might be able to change this to just a `devDependency`, though -- let me review further.

Reply:
Got rid of the peer dependency.